flows/index_from_aggregate_results.py

```diff
@@ -299,7 +300,6 @@ async def create_indexing_summary_artifact(
     )
 
 
-@flow(log_prints=True)
```

Comment:
note: I'd personally be fine with this still being a flow, by removing the other two we'd be at a heathy third of the current levels and could always remove later.

Reply:
✓